deployment/conf/local.cfg

```diff
@@ -3,3 +3,8 @@ META_DIR = ./data/metadata/
 DATA_DIR = ./data/bulk/
 AUTH_URL = https://ci.kbase.us/services/auth/
 CONCIERGE_PATH = /kbaseconcierge
+FILE_EXTENSION_MAPPINGS = ./deployment/conf/supported_apps_w_extensions.json
+DTS_MANIFEST_SCHEMA = ./import_specifications/schema/dts_manifest_schema.json
```

Comment:
These were missing, but only used if a dev runs the service locally (not in Docker).